include/GafferImage/ImageAlgo.inl

```diff
@@ -319,19 +319,13 @@ void parallelProcessTiles( const ImagePlug *imagePlug, const std::vector<std::st
 		const Gaffer::ThreadState &threadState = Gaffer::ThreadState::current();
```

Comment:
My previous comment applies here too : this can simply be `channelDataScope( Context::current() )`. Note also that we don't need the extra `{}` scope to contain it.